runtime/ddr/ddrtable.c

```diff
@@ -33,7 +33,9 @@ extern const J9DDRStructDefinition *getJITAutomatedStructTable(struct OMRPortLib
 extern const J9DDRStructDefinition *getOmrStructTable(struct OMRPortLibrary *portLib);
 extern const J9DDRStructDefinition *getVMStructTable(struct OMRPortLibrary *portLib);
 extern const J9DDRStructDefinition *getStackWalkerStructTable(struct OMRPortLibrary *portLib);
+#ifdef J9VM_OPT_HARMONY
```

Comment:
Please use `#if defined(J9VM_OPT_HARMONY)` instead.

Reply:
Done.  I did not change the existing `#ifdef`s, since we hope they will go away soon-ish.